### PR TITLE
Show resistance exceptions in hazard IWR

### PR DESCRIPTION
**Pull request by CarlosFdez**
image

### CODE REVIEW

File: static/templates/actors/hazard/partials/sidebar.html [hunk 1]
```diff
@@ -89,7 +89,7 @@ <h2>{{localize "PF2E.ImmunitiesLabel"}}</h2>
             </div>
             <div class="tags">
                 {{#each immunities as |immunity slug|}}
-                    <span class="tag" data-slug="{{slug}}">{{localize immunity.label}}</span>
```
Comment (stwlam) [inline static/templates/actors/hazard/partials/sidebar.html:92]: I think workshop uses that?

Comment (CarlosFdez) [reply static/templates/actors/hazard/partials/sidebar.html:92]: Does it? The weakness/resistance ones for hazard were not right at all, and NPC does data-weakness (for all of them....). Should probably make them all the same then. @xdy do you use these slugs, and if so, which one do you use? data-weakness?

Comment (stwlam) [reply static/templates/actors/hazard/partials/sidebar.html:92]: Maybe you can fix 🙂 
https://github.com/foundryvtt/pf2e/issues/3657

Comment (CarlosFdez) [reply static/templates/actors/hazard/partials/sidebar.html:92]: I'll wait to see what the response is, but I'm leaning towards data-slug and data-type.

Comment (xdy) [reply static/templates/actors/hazard/partials/sidebar.html:92]: No, I do it in a createChatMessage hook on a damageroll and then check message.flags?.pf2e?.damageRoll?.types for the attack (or if that's empty, get the original item and check damage.value and get the types) and then compare to traits?.di/dr/dv.value on the target. (In a very messy way. Total rewrite needed.)

Comment (CarlosFdez) [reply static/templates/actors/hazard/partials/sidebar.html:92]: Sorry, we got you mixed up. Up to @shemetz then, but I'm likely going to at least be adding data-slug. Renaming data-weakness might be too annoying to do right now though?

Comment (CarlosFdez) [reply static/templates/actors/hazard/partials/sidebar.html:92]: Changed it to data-weakness="slug", data-resistance="slug", and data-immunity="slug".

Comment (shemetz) [reply static/templates/actors/hazard/partials/sidebar.html:92]: Thanks for notifying me!

**Pull request merged**